Header header
float32 goDirection
float32 goMagnitude
float32 lookDirection
float32 rotate
bool lift
bool land
